packages/website/src/hooks/backend.ts

```diff
@@ -130,21 +120,25 @@ export function useTxnStager(
     ],
   });
 
-  const hashToSign = reads.isSuccess ? (reads.data![0].result as unknown as Address) : null;
-
-  const alreadyStagedSigners = useMemo(() => {
-    if (!hashToSign || !alreadyStaged) {
-      return [];
-    }
-
-    const signers = [];
-    for (const sig of alreadyStaged.sigs) {
-      const regularSig = ethers.utils.arrayify(sig);
-      regularSig[regularSig.length - 1] -= 4;
-      signers.push(ethers.utils.verifyMessage(ethers.utils.arrayify(hashToSign), regularSig));
-    }
+  const hashToSign = reads.isSuccess ? (reads.data![0].result as viem.Address) : null;
+
+  useEffect(() => {
```

Comment:
its really a shame that viem requires `await` for recoverAddress. When I checked the source code, the reason is because it dynamically `import`s a module from inside the recover function. Not sure why they couldn't just let people import this module if they want it.